cli/api/entity_config/config.go

```diff
@@ -52,7 +52,7 @@ func ConfigValueAsBytes(network *net.Network, application, entity, config string
 
 func SetConfig(network *net.Network, application, entity, config, value string) (string, error) {
 	url := fmt.Sprintf("/v1/applications/%s/entities/%s/config/%s", application, entity, config)
-	val := []byte(value)
+	val := []byte("\""+value+"\"")
```

Comment:
This will prevent setting json containers/numbers. Currently you can set both `'{key: 1}'` and `'"mystring"'`.

* Should we add an option which indicates the value is json?
* Do we care about backwards compatibility where someone could already be setting strings with the quotes added?